chalicelib/check_setup.json

```diff
@@ -401,5 +401,22 @@
                 }
             }
         }
+    },
+    "core_project_status":{
+        "title": "Ensure CGAP Core project items are shared",
+        "group": "Metadata checks",
+        "schedule": {
+            "morning_checks": {
+                "cgap": {
+                    "kwargs": {
+                        "primary": true,
+                        "item_type": ["VariantSample", "Case", "Family",
+                            "Individual", "WorkflowRun", "FileFastq",
+                            "FileProcessed", "Sample", "SampleProcessing",
+                            "Report", "QualityMetric", "FilterSet"]
```

Comment:
now that I think about it for some itemtypes like FileProcessed we don't want to change the status if it is 'to be uploaded by workflow' or similar - however the action isn't automatically queued so its probably fine to use discretion in deciding whether to run the action.